src/Microsoft.DotNet.Wpf/specs/tooltip.md

```diff
@@ -0,0 +1,305 @@
+
+
+# Requirements 
+This spec cites the following sources (the [brackets] contain abbreviations used elsewhere in this document):
+
+1. WCAG 2.1, section 1.4.13: [Content on hover or focus](https://www.w3.org/TR/WCAG21/#content-on-hover-or-focus) [WCAG] 
+2. WCAG 2.1:  [Understanding 1.4.13](https://www.w3.org/WAI/WCAG21/Understanding/content-on-hover-or-focus.html)   [WCAG]
+3. MAS 1.4.13:  [Content on hover or focus](https://microsoft.sharepoint.com/:w:/r/sites/accessibility/_layouts/15/Doc.aspx?sourcedoc=%7B56729308-859e-4b32-ab27-04d4364b3003%7D&action=view&wdLOR=c9C9640FB-F04E-401A-BF46-A79887F322D3&wdAccPdf=0&wdparaid=38854747) [MAS]
+4. The [Figma diagram](https://www.figma.com/file/8LeT0Eq7a3ylAyM19c0FJp/ASC%3A-Tooltip?node-id=110%3A0) (as of February 2021) [FD]
+5. Email thread (MS internal) [ET] 
+6. WPF [Tooltip Overview](https://docs.microsoft.com/en-us/dotnet/desktop/wpf/controls/tooltip-overview?view=netframeworkdesktop-4.8) (and related links) [TO]
+
+The first three describe behavior in general terms, guiding any UI platform (WCAG) or Microsoft UI platforms (MAS).
+The Figma diagram has more specific requirements, but is not a complete spec.
+The email thread among many relevant MS teams helped fill in omissions, corrected errors, and clarified ambiguities in the diagram.
+The Tooltip Overview documents the existing behavior in WPF as of .NET 4.7.2.
+
+# Applicability
+
+All the changes described here are required to appear in .NET 6.0.
+
+The behavior changes will also appear in servicing updates for earlier versions as follows:
+- .NET 4.6-4.7.2, .NET 4.8:  the next biannual accessibility update (expected summer 2021)
+- .NET Core 3.1, .NET 5.0:  same time frame as above, patch version TBD
+
+The API changes (and behavior that depends on them) are only in .NET 6.0;
+servicing updates cannot include changes to public API.
+
+In the servicing updates, the new behavior is opt-in by setting
+`Switch.UseLegacyAccessibilityFeatures.5` to false, as described [here](https://docs.microsoft.com/en-us/dotnet/framework/whats-new/whats-new-in-accessibility).
+
+
+
+# Conceptual pages (How To)
+
+## Declaring a tooltip
+You can declare a tooltip in XAML or code as described in [TO].
+Starting in .NET 6.0, your declarations can also use the properties
+`ToolTip.ShowsToolTipOnKeyboardFocus`, and
+`ToolTipService.ShowsToolTipOnKeyboardFocus`.
+
+## Opening a tooltip
+When a parent element declares a tooltip, there are four ways to open (show) the tooltip:
+
+1. Hover the mouse over the parent.
+2. Move keyboard focus to the parent.
+3. Type a keyboard shortcut.
+4. Open the tooltip programmatically.
+
+Unless mentioned otherwise below, the tooltip is shown immediately and its size and position are governed by the placement properties described in [TO].
+
+### Hover
+A hover occurs when the mouse enters the parent element and stays there for a certain time, without any other activity.
```

Comment:
Or touch?

Reply:
Yes.

Reply:
In Xaml, tooltips show with touch while the parent item is pressed. As soon as the user lifts up their finger, the tooltip closes. Similar behavior to mouse hover but requires more effort by the user :) Feels like the correct touch behavior for WPF to use as well.